### ISSUE TITLE
Add message does not allow null values

### ISSUE THREAD

**Issue by floratmin** [connectedcars/gettext-extractor#48]
The `addMessage` method on the `Extractor` class does not allow to add messages where some fields are null. But the definition of the `IMessage` interface allows `null` values for `text`, `textPlural` and `context`.

**Comment by lukasgeiter** [comment]
You're right, that's definitely not consistent. Until this is fixed you can use `undefined` instead of `null`.
